packages/ses/scripts/bundle.js

```diff
@@ -1,5 +1,11 @@
+/* eslint-disable import/first */
 /* global process */
+
 import '../index.js';
+import '../lockdown.js';
+
+lockdown();
+
```

Comment:
This has the effect of ensuring that `lockdown()` gets called before the following imports evaluate:
```suggestion
import '../test/lockdown-safe.js';
```

The proposed change causes `lockdown()` to be called after all imported modules have evaluated. This is one of surprising effects of ESM loader semantics. Imported modules do not evaluate on the stack of the `import` statement in the way that CJS modules _may_ evaluate on the stack of a `require` call. CJS bundlers generally behave like ESM, though.

Even with the above formulation, it is possible to force the subsequent imports to evaluate earlier (and thus in the pre-lockdown realm) by importing this module from another module with a different import order over the same modules. This is outside our concern because this is the entrypoint module for the program.